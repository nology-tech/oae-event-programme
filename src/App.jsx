import { Route, Routes } from "react-router-dom";
import "./App.scss";
import About from "./pages/About/About";
import Home from "./pages/Home/Home";
import Schedule from "./pages/Schedule/Schedule";

const App = () => {
  return (
    <Routes>
      <Route index element={<Home />} />
      <Route path="/our-story" element={<About />} />
      <Route path="/event-schedule" element={<Schedule />} />
      <Route path="/*" element={<Home />} />
    </Routes>
  );
};

export default App;
