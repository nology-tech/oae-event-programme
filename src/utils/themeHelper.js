  const getThemeChange = (theme) => {
    if (typeof theme !== "number") {
      theme = parseInt(theme);
    }
    if(theme === 1) {
      return "primary"
    } else if (theme === 2) {
      return "secondary"
    }
    return "tertiary"
  }    
export default getThemeChange;