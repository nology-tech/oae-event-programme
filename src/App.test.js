import App from "./App";
import { customRender } from "./utils/testUtils";

test("Renders App component with default page", () => {
  const { container } = customRender(<App />);
  expect(container).toMatchSnapshot();
});
