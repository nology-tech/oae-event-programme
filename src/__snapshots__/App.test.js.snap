// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders App component with default page 1`] = `
<div>
  <div>
    <h1>
      Home page
    </h1>
  </div>
</div>
`;
